### ISSUE TITLE
It seems numericality doesn't support negative value

### ISSUE THREAD

**Issue by bibobibo** [ansman/validate.js#191]
I try to use it, then find it doesn't support negative value for numericality in strict mode, am I right?

**Comment by ahaurw01** [comment]
Ran into the same issue. 

https://github.com/ansman/validate.js/blob/master/validate.js#L841

```js
var pattern = "^(0|[1-9]\\d*)";
new RegExp(pattern).test('-5'); // false
```

**Comment by ansman** [comment]
Can't believe I missed that one, thanks.

**Comment by ahaurw01** [comment]
Thanks!

**Comment by ansman** [comment]
A fix has been released in 0.12.0
